### PR TITLE
On commit failure, include topic and partition in error

### PR DESCRIPTION
**Pull request by robot-dreams**

### CODE REVIEW

File: marshal/consumer.go [hunk 1]
```diff
@@ -853,7 +853,10 @@ func (c *Consumer) Commit(msg *Message) error {
 		return cl, ok
 	}()
 	if !ok {
-		return errors.New("Message not committed (partition claim expired).")
+		return fmt.Errorf(
```
Comment (zorkian) [inline marshal/consumer.go]: nit: Too many lines. You can do this in two lines and still obey the line length limit. (I'm allergic to gratuitous vertical space; hard to read when there's one word per line.)

File: marshal/consumer.go [hunk 2]
```diff
@@ -916,7 +919,10 @@ func (c *Consumer) CommitByToken(token CommitToken) error {
...
```
Comment (zorkian) [inline marshal/consumer.go]: nit: Formatting.

**Comment by robot-dreams** [comment]
Sounds good.  BTW as a recovering Java user I am used to being very liberal with newlines, e.g.

    MyBuilder()
        .value(foo)
        .anotherValue(bar)
        .moreValues(baz)
        .areWeDoneYet(bat)
        .nope(0xDEADBEEF)
        .howAboutNow("no")
        .build()

**Comment by zorkian** [comment]
Yeah; formatting is hard. Everybody wants to do it differently. @DrTall and I have had some, ah, spirited discussions about the matter.

**Pull request merged**